R/datatables.R

```diff
@@ -889,7 +889,16 @@ normalizeStyle = function(style) {
   if (is.null(theme)) {
     return('default')
   }
-  style = if ('3' %in% bslib::theme_version(theme)) 'bootstrap' else 'bootstrap4'
+
+  bs_v = as.numeric(bslib::theme_version(theme)[1])
+  # TODO: If DT adds support for BS > 5, update this logic
+  style = if (bs_v > 4)
+      "bootstrap5"
+    else if (bs_v > 3)
+      "bootstrap4"
+    else
+      "bootstrap"
```

Comment:
```suggestion
  style = paste0("bootstrap", if (bs_v > 3) bs_v)
```

Reply:
I didn't do this because I'm not familiar with the process for updating the Bootstrap-specific styles in DT. I was imagining that if/when `bslib` begins offering Bootstrap 6, setting `style = "bootstrap6"` might be worse than using the `"bootstrap5"` styles, especially if Bootstrap 6 support in DT requires adding new CSS files to DT.

If a manual update is required, I'd personally leave this code (with all of its code smell) to avoid giving the false impression that Bootstrap 6 is automatically supported.

I defer to you though, since I haven't investigated how dependencies in DT are handled.

Reply:
Okay, that makes sense to me. Thank you!